docs/Makefile

```diff
@@ -14,7 +14,11 @@ ALLSPHINXOPTS   = -d $(BUILDDIR)/doctrees $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
 # the i18n builder cannot share the environment and doctrees with the others
 I18NSPHINXOPTS  = $(PAPEROPT_$(PAPER)) $(SPHINXOPTS) .
 
-.PHONY: help clean html dirhtml singlehtml pickle json htmlhelp qthelp devhelp epub latex latexpdf text man changes linkcheck doctest gettext coverage
+GENKEYIMG_SCRIPT = ../scripts/gen-keybinding-img -o ./_static/keybindings
```

Comment:
```suggestion
GENKEYIMG_SCRIPT = ../scripts/gen-keybinding-img
```

Reply:
the directory is hardcoded and later it is used in other scripting parts (next lines). Why it would be  better to move? If user will want to have its own images, he will run script and not `make`

Reply:
It felt more logical to me, but now that I'm looking at it again, why not just put everything into the `genkeyimage` task? I don't see why we would want these three lines to be variables of the Makefile.

Reply:
moved under `genkeyimage:`